deploy.sh

```diff
@@ -18,23 +18,28 @@
 ROOT_DIR="${PWD}"
 SCRIPTS_DIR="${ROOT_DIR}/scripts"
 
-AUTOTUNE_OPERATOR_CRD="manifests/autotune-operator-crd.yaml"
-AUTOTUNE_CONFIG_CRD="manifests/autotune-config-crd.yaml"
-AUTOTUNE_QUERY_VARIABLE_CRD="manifests/autotune-query-variable-crd.yaml"
-AUTOTUNE_DEPLOY_MANIFEST_TEMPLATE="manifests/autotune-operator-deployment.yaml_template"
-AUTOTUNE_DEPLOY_OPENSHIFT_MANIFEST_TEMPLATE="manifests/autotune-operator-openshift-deployment.yaml_template"
-AUTOTUNE_DEPLOY_MANIFEST="manifests/autotune-operator-deployment.yaml"
-AUTOTUNE_RB_MANIFEST_TEMPLATE="manifests/autotune-operator-rolebinding.yaml_template"
-AUTOTUNE_RB_MANIFEST="manifests/autotune-operator-rolebinding.yaml"
-AUTOTUNE_ROLE_MANIFEST="manifests/autotune-operator-role.yaml"
-AUTOTUNE_SA_MANIFEST="manifests/autotune-operator-sa.yaml"
-SERVICE_MONITOR_MANIFEST="manifests/servicemonitor/autotune-service-monitor.yaml"
+AUTOTUNE_DIR="./manifests/autotune"
+MONITORING_DIR="./manifests/monitoring"
```

Comment:
As I mentioned yesterday this should be `crc` not monitoring

Reply:
Also change MONITORING_DIR to `CRC_DIR`

Reply:
Done